lib/gui/modules/analytics.js

```diff
@@ -164,7 +88,7 @@ analytics.service('AnalyticsService', function($log, $window, $mixpanel, Setting
    * @public
    *
    * @description
-   * This function logs an exception in TrackJS.
+   * This function logs an exception in Sentry.
```

Comment:
Again, lets say something generic like "this function logs an exception to an error reporting service" or something like that.